patches/chrome-common-extensions-api-_permission_features.json.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/common/extensions/api/_permission_features.json b/chrome/common/extensions/api/_permission_features.json
+index 2a5dd641288bc49ce33bd1e2ab08e118ecad1446..2ac1ce93c569a7ce8a208172606c1edeb4ce4f48 100644
+--- a/chrome/common/extensions/api/_permission_features.json
++++ b/chrome/common/extensions/api/_permission_features.json
+@@ -667,6 +667,7 @@
+     "whitelist": [
+       "63ED55E43214C211F82122ED56407FF1A807F2A3",  // Media Router Dev
+       "226CF815E39A363090A1E547D53063472B8279FA"   // Media Router Release
++      , "A321D47A2B4CA86898167A55CA8B2E02385EA7CD" // Brave Shields
```

Comment:
@bridiver there's a few ways to approach this:

1) Keep this .patch of _**permission**_features.json
2) _Overwrite_ the settingsPrivate entry in _**api**_features.json using the same technique we're doing for other APIs. Disadvantage: requires duplicating security-focused chromium code and is a risk IMO.
3) Override _permission_features.json in the same way we override _api_features.json. Comes with same risk as 2)?.
4) Invent a new way to inject additions to the JSON of _api_features.json or _permission_features.json without overwriting the chromium source.

(Preferrably as a follow-up to unblock @cezaraugusto 's feature in this PR).

Reply:
see brave/common/extensions/api/_api_features.json

Reply:
oh, this is permissions, but easy to copy it

Reply:
this is pretty trivial change so I think we should fix before merging

Reply:
@bridiver added _api_permissions.json, please review